cpp/json_ffi/image_utils.h

```diff
@@ -0,0 +1,29 @@
+/*!
+ *  Copyright (c) 2023 by Contributors
+ * \file json_ffi/image_utils.h
+ * \brief The header of Image utils for JSON FFI Engine in MLC LLM.
+ */
+#ifndef MLC_LLM_JSON_FFI_IMAGE_UTILS_H_
+#define MLC_LLM_JSON_FFI_IMAGE_UTILS_H_
+
+#include <tvm/runtime/ndarray.h>
+
+#include <string>
+
+namespace mlc {
+namespace llm {
+namespace json_ffi {
+
+using namespace tvm::runtime;
+
+unsigned char* LoadImageFromBase64(std::string base64_str, int* width, int* height,
```

Comment:
We generally prefer managed memory. We can consider return an NDArray (on CPU), with {height, width} dimension and `uint8` as dtype.